CHANGELOG/CHANGELOG-1.9.md

```diff
@@ -0,0 +1,74 @@
+
+### Added
+
+- Add the following capabilities to the Multi-cluster feature:
+  * Add support for Pod to Pod connectivity across clusters. ([#4219](https://github.com/antrea-io/antrea/pull/4219), [@hjiajing])
+  * Add active-passive mode high availability support for Gateway Nodes. ([#4069](https://github.com/antrea-io/antrea/pull/4069), [@luolanzone])
+  * Allow Pod IPs as Endpoints of Multi-cluster Service; a field `endpointIPType` is added to the MultiClusterConfig API to specify the IPs of Endpoints. ([#4198](https://github.com/antrea-io/antrea/pull/4198), [@luolanzone])
+  * Add `antctl mc get joinconfig` command to print ClusterSet join parameters. ([#4299](https://github.com/antrea-io/antrea/pull/4299), [@jianjuns])
+  * Add `antctl mc get|delete membertoken` commands to get/delete member token. ([#4254](https://github.com/antrea-io/antrea/pull/4254), [@bangqipropel])
+- Add rule name to Audit Logging for Antrea-native policies. ([#4178](https://github.com/antrea-io/antrea/pull/4178), [@qiyueyao])
+- Add Service health check similar to kube-proxy in antrea-agent; it provides HTTP endpoints "http://NODE-IP:healthCheckNodePort/healthz" for querying number of local Endpoints of a Service. ([#4120](https://github.com/antrea-io/antrea/pull/4120), [@shettyg])
+- Add S3Uploader as a new exporter of Flow Aggregator, which periodically exports expired flow records to AWS S3 storage bucket. ([#4143](https://github.com/antrea-io/antrea/pull/4143), [@heanlan])
+
+### Changed
+
+- AntreaProxy now supports more than 800 Endpoints for a Service. ([#4167](https://github.com/antrea-io/antrea/pull/4167), [@hongliangl])
+- Add OVS connection check to Agent's liveness probes for self-healing on OVS disconnection. ([#4126](https://github.com/antrea-io/antrea/pull/4126), [@tnqn])
+- Startup scripts now perform cleanup automatically upon non-Kubernetes Nodes (ExternalNode) restart before running antrea-agent. ([#4277](https://github.com/antrea-io/antrea/pull/4277), [@Anandkumar26])
+- Make tunnel csum option configurable and default to false which avoids double encapsulation checksum issues on some platforms. ([#4250](https://github.com/antrea-io/antrea/pull/4250), [@tnqn])
+- Use standard value type for k8s.v1.cni.cncf.io/networks annotation. ([#4146](https://github.com/antrea-io/antrea/pull/4146), [@antoninbas])
+- Update Go to v1.19. ([#4106](https://github.com/antrea-io/antrea/pull/4106), [@antoninbas])
+- Publish scripts and binaries needed for running Antrea on VMs (ExternalNode) in release assets. ([#4266](https://github.com/antrea-io/antrea/pull/4266) [#4113](https://github.com/antrea-io/antrea/pull/4113), [@antoninbas] [@Anandkumar26])
+- Manifest files in the main branch switches to the docker.io registry for "latest" antrea images. ([#4235](https://github.com/antrea-io/antrea/pull/4235), [@antoninbas])
+- Add API support for reporting Antrea NetworkPolicy realization failure. ([#4248](https://github.com/antrea-io/antrea/pull/4248), [@wenyingd])
```

Comment:
Is this feature complete? If so, it should be in "Added".

Reply:
It's not a complete feature but an API change needed by other components to report failure, i.e. Nephe and antrea-agent.

Reply:
Ok